bin/themes/Dark/style.css

```diff
@@ -89,502 +89,500 @@ QCheckBox::indicator:checked:pressed,
-    border: 1px solid transparent;
-    /* reserve space for selection border */
+  padding: 5px 30px 5px 30px;
+  border: 1px solid transparent;
+  /* reserve space for selection border */
 }
 
 QMenu::item:selected {
-    color: #e0e0e0;
+  color: #e0e0e0;
 }
 
 QMenu::separator {
-    height: 2px;
-    background: lightblue;
-    margin-left: 10px;
-    margin-right: 5px;
+  height: 2px;
+  background: lightblue;
+  margin-left: 10px;
+  margin-right: 5px;
 }
 
 QMenu::indicator {
-    width: 18px;
-    height: 18px;
+  width: 18px;
+  height: 18px;
 }
 
-
 /* non-exclusive indicator = check box style indicator
    (see QActionGroup::setExclusive) */
 
 QMenu::indicator:non-exclusive:unchecked {
-    image: url(./rc/checkbox_unchecked.png);
+  image: url(./rc/checkbox_unchecked.png);
 }
 
 QMenu::indicator:non-exclusive:unchecked:selected {
-    image: url(./rc/checkbox_unchecked_disabled.png);
+  image: url(./rc/checkbox_unchecked_disabled.png);
 }
 
 QMenu::indicator:non-exclusive:checked {
-    image: url(./rc/checkbox_checked.png);
+  image: url(./rc/checkbox_checked.png);
 }
 
 QMenu::indicator:non-exclusive:checked:selected {
-    image: url(./rc/checkbox_checked_disabled.png);
+  image: url(./rc/checkbox_checked_disabled.png);
 }
 
-
 /* exclusive indicator = radio button style indicator (see QActionGroup::setExclusive) */
 
 QMenu::indicator:exclusive:unchecked {
-    image: url(./rc/radio_unchecked.png);
+  image: url(./rc/radio_unchecked.png);
 }
 
 QMenu::indicator:exclusive:unchecked:selected {
-    image: url(./rc/radio_unchecked_disabled.png);
+  image: url(./rc/radio_unchecked_disabled.png);
 }
 
 QMenu::indicator:exclusive:checked {
-    image: url(./rc/radio_checked.png);
+  image: url(./rc/radio_checked.png);
 }
 
 QMenu::indicator:exclusive:checked:selected {
-    image: url(./rc/radio_checked_disabled.png);
+  image: url(./rc/radio_checked_disabled.png);
 }
 
 QMenu::right-arrow {
-    margin: 5px;
-    image: url(./rc/right_arrow.png)
+  margin: 5px;
+  image: url(./rc/right_arrow.png);
 }
 
 QWidget:disabled {
-    color: #454545;
-    background-color: #212121;
+  color: #454545;
+  background-color: #212121;
 }
 
 QAbstractItemView {
-    alternate-background-color: #212121;
-    color: #e0e0e0;
-    border: 1px solid #212121;
-    border-radius: 2px;
+  alternate-background-color: #212121;
+  color: #e0e0e0;
+  border: 1px solid #212121;
+  border-radius: 2px;
 }
 
 QWidget:focus,
 QMenuBar:focus {
-    border: 1px solid #414141;
+  border: 1px solid #414141;
 }
 
 QTabWidget:focus,
 QCheckBox:focus,
 QRadioButton:focus,
 QSlider:focus {
-    border: none;
+  border: none;
 }
 
 QLineEdit {
-    background-color: #313131;
-    padding: 1px;
-    border-style: solid;
-    border: 1px solid #212121;
-    border-radius: 2px;
-    color: #e0e0e0;
+  background-color: #313131;
+  padding: 1px;
+  border-style: solid;
+  border: 1px solid #212121;
+  border-radius: 2px;
+  color: #e0e0e0;
+  font-size: 20px;

```

Comment:
I'm not sure if you should change all the `QLineEdit`. Perhaps instead set the font size in the calculator directly...